harmony_netcdf_to_zarr/convert.py

```diff
@@ -152,6 +147,7 @@ def _output_worker(output_queue: Queue, shared_namespace: Namespace,
         array.
 
     """
+    processed_granules = 0
```

Comment:
This is definitely an improvement that having 1500 identical log messages, nice. Did you try using a shared value between the processes using something like [multiprocessing.Value](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Value) - that might kick it up another notch.

Reply:
Yeah, good idea. [4c479ca](https://github.com/nasa/harmony-netcdf-to-zarr/pull/39/commits/4c479ca303559c3a4e8121ba5cf6c51344274ab8)

Reply:
Thanks for the update!